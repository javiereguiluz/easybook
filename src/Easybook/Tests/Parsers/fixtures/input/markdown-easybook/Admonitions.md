# Admonitions Lorem ipsum dolor sit amet #

Consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et
dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
ullamco laboris nisi ut aliquip ex ea commodo consequat.

> Duis aute irure dolor in reprehenderit in voluptate velit esse
> cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
> cupidatat non.

> Duis aute irure dolor in reprehenderit in voluptate velit esse
> cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
> cupidatat non.

Consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et
dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
ullamco laboris nisi ut aliquip ex ea commodo consequat.

A> Consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et
A> dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
A> ullamco laboris nisi ut aliquip ex ea commodo consequat.

A> Consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.

Consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et
dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
ullamco laboris nisi ut aliquip ex ea commodo consequat.

N> Consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et
N> dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
N> ullamco laboris nisi ut aliquip ex ea commodo consequat.

I> Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod.
I> Tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.

## Admonitions Consectetur adipisicing elit ##

N> Consectetur **adipisicing elit**, sed do eiusmod tempor incididunt ut labore et
N> dolore magna aliqua. Ut *enim ad minim* veniam, quis nostrud exercitation
N> ullamco laboris [nisi ut aliquip](http://easybook-project.org) ex ea commodo consequat.

T> Consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et
T> dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
T> ullamco laboris nisi ut aliquip ex ea commodo consequat.
T> > Lorem ipsum dolor sit amet, consectetur adipisicing elit
T> > sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
T> > Ut enim ad minim veniam, quis nostrud exercitation ullamco
T> > laboris nisi ut aliquip ex ea commodo consequat.
T> Duis aute irure dolor in reprehenderit in voluptate velit esse cillum.

Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat.

W> Duis aute irure dolor in reprehenderit in voluptate velit esse
W> cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
W> proident, sunt in culpa qui officia deserunt mollit anim id est laborum:
W> 
W>   * Lorem ipsum dolor sit amet.
W>   * Consectetur adipisicing elit, **sed do eiusmod**.
W>   * Tempor incididunt ut *labore et dolore* magna aliqua.
W> 
W> Ut enim ad minim veniam.

Quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur.

E> Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia.
E> Deserunt mollit anim id est laborum.

I> Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
I> tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,

Q> quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
Q> consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse

D> cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
D> proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
