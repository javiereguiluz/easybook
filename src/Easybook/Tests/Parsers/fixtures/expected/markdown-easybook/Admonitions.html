<h1 id="admonitions-lorem-ipsum-dolor-sit-amet">Admonitions Lorem ipsum dolor sit amet</h1>

<p>Consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et
dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>

<blockquote>
  <p>Duis aute irure dolor in reprehenderit in voluptate velit esse
  cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
  cupidatat non.</p>
  
  <p>Duis aute irure dolor in reprehenderit in voluptate velit esse
  cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
  cupidatat non.</p>
</blockquote>

<p>Consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et
dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>

<div class="admonition aside">
  <p>Consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et
  dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
  ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
</div>

<div class="admonition aside">
  <p>Consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
</div>

<p>Consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et
dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>

<div class="admonition note">
  <p>Consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et
  dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
  ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
</div>

<div class="admonition information">
  <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod.
  Tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.</p>
</div>

<h2 id="admonitions-consectetur-adipisicing-elit">Admonitions Consectetur adipisicing elit</h2>

<div class="admonition note">
  <p>Consectetur <strong>adipisicing elit</strong>, sed do eiusmod tempor incididunt ut labore et
  dolore magna aliqua. Ut <em>enim ad minim</em> veniam, quis nostrud exercitation
  ullamco laboris <a href="http://easybook-project.org">nisi ut aliquip</a> ex ea commodo consequat.</p>
</div>

<div class="admonition tip">
  <p>Consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et
  dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
  ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
  
  <blockquote>
    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit
    sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
    Ut enim ad minim veniam, quis nostrud exercitation ullamco
    laboris nisi ut aliquip ex ea commodo consequat.
    Duis aute irure dolor in reprehenderit in voluptate velit esse cillum.</p>
  </blockquote>
</div>

<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat.</p>

<div class="admonition warning">
  <p>Duis aute irure dolor in reprehenderit in voluptate velit esse
  cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
  proident, sunt in culpa qui officia deserunt mollit anim id est laborum:</p>
  
  <ul>
  <li>Lorem ipsum dolor sit amet.</li>
  <li>Consectetur adipisicing elit, <strong>sed do eiusmod</strong>.</li>
  <li>Tempor incididunt ut <em>labore et dolore</em> magna aliqua.</li>
  </ul>
  
  <p>Ut enim ad minim veniam.</p>
</div>

<p>Quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur.</p>

<div class="admonition error">
  <p>Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia.
  Deserunt mollit anim id est laborum.</p>
</div>

<div class="admonition information">
  <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
  tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,</p>
</div>

<div class="admonition question">
  <p>quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
  consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse</p>
</div>

<div class="admonition discussion">
  <p>cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
  proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
</div>
